lib/webauthn/authenticator_data/attested_credential_data.rb

```diff
@@ -26,10 +26,14 @@ def valid?
         data.length >= AAGUID_LENGTH + ID_LENGTH_LENGTH && valid_credential_public_key?
       end
 
-      def aaguid
+      def raw_aaguid
         data_at(0, AAGUID_LENGTH)
       end
 
+      def aaguid
+        raw_aaguid.unpack("H8H4H4H4H12").join("-")
```

Comment:
For the record, and let me know if I am wrong, this is GUID format per [RFC4122](https://tools.ietf.org/html/rfc4122).

Despite W3C Rec not explicitly telling us to interpret AAGUID like this, there are FIDO2 docs (https://fidoalliance.org/specs/fido-v2.0-ps-20150904/fido-key-attestation-v2.0-ps-20150904.html) saying AAGUID should be a "version 4 GUID").

Reply:
Thanks for providing sources 👍